modules/programs/vscode.nix

```diff
@@ -211,12 +220,23 @@ in {
           else
             builtins.attrNames (builtins.readDir (ext + "/${subDir}")));
       in if cfg.mutableExtensionsDir then
-        mkMerge (concatMap toPaths cfg.extensions)
+        mkMerge (concatMap toPaths cfg.extensions ++ lib.optional (lib.version >= "1.74.0") [{
```

Comment:
```suggestion
        mkMerge (concatMap toPaths cfg.extensions ++ lib.optional (lib.versionAtLeast vscodeVersion "1.74.0") [{
```

Reply:
I tried to find that function in `nixpkgs` and I just couldn't (and I couldn't remember its name). I didn't think to look in `strings.nix` at all!